actions/src/dispatcher.js

```diff
@@ -31,8 +31,10 @@ module.exports = {
     const repoName = context.payload.repository.name.toLowerCase();
     const checksWhitelist = constants.getChecksWhitelist();
     if (Object.prototype.hasOwnProperty.call(checksWhitelist, repoName)) {
+      core.info('Repo is whitelisted');
```

Comment:
Do we need to add this? Ditto below.

Reply:
Yes, It makes debugging a bit easier for future developers when introducing new actions.

Reply:
Ok.